### DISCUSSION TITLE
500 server error when generating large dataset PDF

### DISCUSSION THREAD

**Discussion by meetings03** [code4recovery/pdf#69] [General]
We had an issue in the past with trying to generate a pdf with around 3000 meeting listed. We get a 500 server error. The issue was resolved at one point but now we are experiencing the same issue again.
The pdf is for a state wide printed meeting list that we use for H&I facilities and prison release packages.
The last time it happened I believe you determined it was an issue with overloading the code4recovery servers while generating a pdf from the large amount of data. We're running WP 6.4.3. TSML plugin 3.14.32 on area03.com.

**Comment by anchovie91471** [comment]
Can I ask what the use case is for printing a list so large? It seems like a single list with meetings hundreds of miles apart wouldn't be that useful to someone trying to find a meeting.

Perhaps we could help you break the list down into smaller regions?

**Comment by meetings03** [comment]
The primary reason we do a statewide list is that they get distributed 
around the state to various institutions such as treatment centers and 
prisons to which the the discharge/releasee may or may not be 
re-entering the local community in which they are being 
discharged/released.
They also may not have access to, or know how to fully utilize a digital 
device to immediately find meetings. The list is a resource that gives 
them a way to find meetings in whatever community they end up in after 
discharge or release.
As an example, Valley Wide H&I which encompasses most the Phoenix metro 
area distributes around 1200 of them per year and other various H&I, 
Bridging the Gap and Corrections Committees around the state go through 
just as many.

One of the reasons we really liked the TSML plugin was the capability of 
a novice level Meeting List Coordinator to maintain the list easily 
online, AND be able to generate the lists for printing without needing 
to have an advanced level of knowledge, or software, to be able compile 
the list in a print ready format.

It worked beautifully while it was working.
We are currently out of the printed lists and I have orders for more 
from various entities and need to get another printing going ASAP.
Hopefully this helps clarify our need to be able to generate the 
statewide list for Arizona meetings.

Thanks,
Greg S.
Area 03 Meeting List Coordinator

**Comment by anchovie91471** [comment]
Can you send the feed url you're using to generate with? I can probably just generate one for you to deal with your immediate need. Meantime, we can discuss options internally for handling this. Being that we are not for profit, we have to work with our limited server resources.

Thanks!

**Comment by anchovie91471** [comment]
We found that the memory limit increase we had put in place a while back was reverted when we changed PHP versions. We have fixed the issue. Can you try your feed again and see if you see an error?
